File: anglee-dashboard-core/models/movie.js
const mongoose = require("mongoose");
const Schema = mongoose.Schema;

const movieSchema = new Schema({
  title: {
    type: String,
    require: true
  },
  releaseDate: {
    type: Date,
    require: true
  },
  cast : Map,
  imageUrl: {
    type: String,
    require: true
  },
  description: {
    type: String,
    require: true
  }
});

module.exports = mongoose.model('Movie', movieSchema);

File: anglee-dashboard-spring/src/main/java/com/aieener/anglee/dashboard/dataobject/Movie.java
package com.aieener.anglee.dashboard.dataobject;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Id;
import java.util.Date;

@Entity
@Data
public class Movie {
  @Id
  private String title;
  private Date releaseDate;
  private String imageUrl;
  private String imdbReviewId;
}
